html/en/cloud/saas-release/snapshot-restore.html

```diff
@@ -1177,4 +1177,4 @@ <h3><a href="/about">About ></a></h3>
 
             </body>
         
-</html>
```

Comment:
I'm not sure why this line has been changed. Looks like getting rid of special char when edit file. I tried to edit in github UI and also vi editor on mac terminal.